docusaurus/docs/develop/path/cheat_sheet_morse.md

```diff
@@ -0,0 +1,208 @@
+
+:::caution
+
+This is a manual and poorly documented process in Morse.
+
+:::
+
+`Application Authentication Tokens` (**AATs**) are auth tokens that allow application
+clients to access the network without the need to expose their private keys.
+Once you have one or more valid AATs, you can populate the configuration files required to run the full `PATH Gateway` instance.
+
+The following resources are also good references and starting points:
+
+- [What are AATs?](https://docs.pokt.network/gateways/host-a-gateway/relay-process#what-are-aats)
+- [Host a Gateway on Morse](https://docs.pokt.network/gateways/host-a-gateway)
+- [pocket-core/doc/specs/application-auth-token.md](https://github.com/pokt-network/pocket-core/blob/7f936ff7353249b161854e24435e4bc32d47aa3f/doc/specs/application-auth-token.md)
+- [pocket-core/doc/specs/cli/apps.md](https://github.com/pokt-network/pocket-core/blob/7f936ff7353249b161854e24435e4bc32d47aa3f/doc/specs/cli/apps.md)
+- [Gateway Server Kit instructions (as a reference)](https://github.com/pokt-network/gateway-server/blob/main/docs/quick-onboarding-guide.md#5-insert-app-stake-private-keys)
+
+_If you are unsure of where to start, you should reach out to the team directly._
+
+### 1.1 AAT Generation
+
+We strongly recommend following the resources above.
+
+However, assuming you have access to a **staked application**, you can follow the instructions below.
+
+<details>
+
+<summary>tl;dr Use at your own risk copy-pasta commands</summary>
+
+**Get the source code:**
+
+```bash
+git clone git@github.com:pokt-network/pocket-core.git
+cd pocket-core
+```
+
+**Build your own `pocket-core` binary:**
+
+```bash
+go build -o pocket ./app/cmd/pocket_core/main.go
+```
+
+**Generate an AAT:**
+
+```bash
+./pocket-core create-aat <ADDR_APP> <CLIENT_PUB>
+```
+
+**Take note of the output:**
+
+```json
+{
+  "version": "0.0.1",
+  "app_pub_key": <APP_PUB>,
+  "client_pub_key": <CLIENT_PUB>,
+  "signature": <APP_SIG>
+}
+```
+
+**So you can prepare a configuration like so:**
+
+```yaml
+morse_config:
+  # ...
+  relay_signing_key: "CLIENT_PRIV"
+  # ...
+signed_aats:
+  <ADDR_APP>:
+    client_public_key: "<CLIENT_PUB>"
+    application_public_key: "<APP_PUB>"
+    application_signature: "<APP_SIG>"
+```
+
+</details>
+
+## 2. Configure PATH
+
+### 2.1 Generate Morse Config
+
+Run the following commands to generate a Morse config at `local/path/config/.config.yaml`:
+
+```bash
+make prepare_morse_e2e_config # Generate ./e2e/.morse.config.yaml
+make copy_morse_e2e_config_to_local # Copy to ./local/path/config/.config.yaml
+# Manually update ./local/path/config/.config.yaml
+```
+
+### Update & Verify the Configuration
+
+You'll need to manually update these fields in the config file:
+
+- `url`
+- `relay_signing_key`
+- `signed_aats`
+
+And then check the updated config:
+
+```bash
+cat local/path/config/.config.yaml
+```
+
+## 3. Start PATH
+
+Run the entire stack (PATH, Envoy, Auth Server) by running:
```

Comment:
This guide does not cover the process of generating and populating the Envoy configuration files, which is necessary to run the entire stack in Tilt. 

It also doesn't cover installing the required dependencies for running envoy (eg. Tilt, kind, helm, etc).

So we'll need to add a link to the Envoy configuration or a short section inside this walkthrough.

Reply:
Going to reply in one "mega comment"